beta/src/content/learn/start-a-new-react-project.md

```diff
@@ -46,15 +46,18 @@ npm start
 
 </TerminalBlock>
 
-For more information, [check out the official guide.](https://create-react-app.dev/docs/getting-started)
-
-> Create React App doesn't handle backend logic or databases. You can use it with any backend. When you build a project, you'll get a folder with static HTML, CSS and JS. Because Create React App can't take advantage of the server, it doesn't provide the best performance. If you're looking for faster loading times and built-in features like routing and server-side logic, we recommend using a framework instead.
+For more information, [check out the official Vite docs.](https://vitejs.dev/)
 
 ### Popular alternatives {/*toolkit-popular-alternatives*/}
 
-* [Vite](https://vitejs.dev/guide/)
+Vite is not the only option for bundling your React-based single page applications. Here are some other popular options:
+
+* [create-react-app](https://create-react-app.dev/)
```

Comment:
it's a dead project. it should either be explicitly marked as such, or not mentioned at all

Reply:
![image](https://user-images.githubusercontent.com/16005493/214734597-c0eb7d20-bac4-4997-972e-bda6c4d1e8b8.png)

Linking incase people didn't know:

[CRA](https://github.com/facebook/create-react-app)